### DISCUSSION TITLE
Looking for contributions in Telegram Bot/Mini App specifically for Kaia DApps

### DISCUSSION THREAD

**Discussion by zxstim** [kaiachain/kaia-dapp-mono#217] [Requests]
- Things like Telegram bot/mini app templates, guides, tutorials to help other developers
- Improvements with onchain interactions within telegram (embedded wallets, good WalletConnect integration, etc.) for Kaia DApps, wallets, projects

Goal is helping Kaia DApps to grow more on Telegram (right now Telegram support TON way more with their in-app wallet)

Note:
- Not looking for chat bot

**Comment by Abidoyesimze** [comment]
already working on it
![Screenshot from 2024-11-05 01-00-39](https://github.com/user-attachments/assets/abef31b5-90e8-4fe2-add8-56c4674694dd)

**Reply by Abidoyesimze** [reply]
![Screenshot from 2024-11-05 01-21-23](https://github.com/user-attachments/assets/4ab73cfb-b443-4d09-9726-9a7fc5055126)

**Reply by zxstim** [reply]
hmmm... not really this tho. gonna clarify more

**Reply by zxstim** [reply]
updated the original request for clarity. Sorry, not looking for chat bot.

**Reply by Abidoyesimze** [reply]
Hi there so i have create the nft bot and create an issue but not approved yet

**Reply by Wake-cloud** [reply]
Would you like any contributions ? please feel free to ask me !

**Reply by zxstim** [reply]
> Hi there so i have create the nft bot and create an issue but not approved yet

just got back from devcon to review the submission. you will see an update soon

**Reply by zxstim** [reply]
> Would you like any contributions ? please feel free to ask me !

make a proposal, follow the guideline here:  
https://dorahacks.io/hackathon/kaia-fall-odyssey/detail#submission-guidelines

**Comment by Wake-cloud** [comment]
Hello there i can help you with your bots. Just fill me in what you need trading bot ? pump.fun bot ? moonshotbot ? just let me know!

**Reply by kaia-dev-intern** [reply]
go here https://dorahacks.io/hackathon/kaia-fall-odyssey/detail#submission-guidelines

**Comment by Abidoyesimze** [comment]
Pls you can still hold on for it I'm trying to add some features to the bot
because you said I should add some scopes which is not there before so
already working on it

**Comment by Abidoyesimze** [comment]
You can check now it's been deployed
